lib/exercism/rikki.rb

```diff
@@ -1,19 +1,27 @@
 require 'digest/sha1'
 
-class Rikki
-  def self.validate(key)
+module Rikki
+  extend self
```

Comment:
I don't have super strong feelings about class vs module. We don't instantiate anything so I'd be fine with a module. I prefer to have the `self`s be explicit, just because it's confusing to me otherwise.